src/test-helpers/deleteThoughtAtFirstMatch.ts

```diff
@@ -19,6 +19,8 @@ const getThoughtAndParentPath = (state: State, at: string[]): [Thought, Path] =>
 
   const thought = pathToThought(state, path)
 
+  if (!thought) throw new Error(`Thought not found for path: ${path}`)
```

Comment:
it's a test file, and we throw an error for empty path above, so thought why not